app/controllers/ctc/questions/confirm_payment_controller.rb

```diff
@@ -0,0 +1,29 @@
+module Ctc
+  module Questions
+    class ConfirmPaymentController < QuestionsController
+      include AuthenticatedCtcClientConcern
+
+      layout "intake"
+
+      def edit
+        @ctc_amount = ChildTaxCreditCalculator.total_advance_payment_2021(current_intake.dependents)
```

Comment:
Either the ChildTaxCreditCalculator OR the data that we pass in needs to be restricted to only qualifying children.

I suggest passing in all of the dependents on the intake as you are doing right now, and then narrowing it down to only dependents that respond `true` to `dependent.qualifying_child_2020?`

It may make your life easier to add a method to TaxReturn to make this calculation so that we can ensure that the interface is consistent as well as ensuring that people can't get an unexpected/incorrect result by passing in the incorrect set of dependents.